### ISSUE TITLE
#option to exclude curves from import

### ISSUE THREAD

**Issue by lfertig** [jesterKing/import_3dm#72]
It should be possible to turn off curves when importing a file, as they can massively impact import time! On a 22mb test file that had several section view blocks saved alongside the 3d geometry, import improved from ~10minutes to a minute or so when getting rid of all curves.

This will probably leave many empty block definitions (all 2d-only blocks), which should in turn also be excluded to not clutter up the instance definition collections.

**Comment by tsvilans** [comment]
True, why don't we add a filter for all object types? They could all be on by default, but then you have the option of importing only what you need.
